test/utils/light-client.ts

```diff
@@ -0,0 +1,238 @@
+import type { SyncCommitteeFast } from "@lodestar/light-client";
+import type {
+    SnowbridgeBeaconPrimitivesBeaconHeader,
+    SnowbridgeBeaconPrimitivesDenebExecutionPayloadHeader,
+    SnowbridgeBeaconPrimitivesExecutionProof,
+    SnowbridgeBeaconPrimitivesUpdatesCheckpointUpdate,
+    SnowbridgeBeaconPrimitivesVersionedExecutionPayloadHeader,
+    SnowbridgeCoreInboundMessage,
+    SnowbridgeCoreInboundProof,
+} from "@polkadot/types/lookup";
+import type { ApiPromise } from "@polkadot/api";
+import { u8aToHex } from "@polkadot/util";
+import type { H256 } from "@polkadot/types/interfaces";
+import type { Bytes, Vec } from "@polkadot/types-codec";
+import { TransactionType } from "@ethereumjs/tx";
+
+export const BLOCKS_ROOTS_ROOT_GINDEX = 37;
+
+export const CURRENT_SYNC_COMMITTEE_GINDEX = 54;
+
+function defaultBeaconBlockHeader(slot: Slot): deneb.LightClientHeader {
+    const header = ssz.deneb.LightClientHeader.defaultValue();
+    header.beacon.slot = slot;
+    return header;
+}
+
+function createSyncCommittee(seed: number) {
+    const skBytes: Buffer[] = [];
+    for (let i = seed; i < seed + SYNC_COMMITTEE_SIZE; i++) {
+        const buffer = Buffer.alloc(32, 0);
+        buffer.writeInt16BE(i + 1, 30); // Offset to ensure the SK is less than the order
+        skBytes.push(buffer);
+    }
+    const sks = skBytes.map((skBytes) => bls.SecretKey.fromBytes(skBytes));
+    const pks = sks.map((sk) => sk.toPublicKey());
+    const pubkeys = pks.map((pk) => pk.toBytes());
+
+    const syncCommittee: altair.SyncCommittee = {
+        pubkeys,
+        aggregatePubkey: bls.aggregatePublicKeys(pubkeys),
+    };
+
+    const syncCommitteeFast: SyncCommitteeFast = {
+        pubkeys: pks,
+        aggregatePubkey: bls.PublicKey.fromBytes(bls.aggregatePublicKeys(pubkeys)),
+    };
+
+    return {
+        syncCommittee,
+        syncCommitteeFast,
+        sks,
+    };
+}
+
+async function createReceiptTrie(
+    contractAddress: Uint8Array,
+    topics: Array<Uint8Array>,
+    messages: Array<Uint8Array>,
+    logsBloom: Uint8Array
+) {
+    const logs = [];
+    for (const message of messages) {
+        const log: Log = [contractAddress, topics, message];
+        logs.push(log);
+    }
+
+    const trie = await Trie.create();
+    const receipt: PostByzantiumTxReceipt = {
+        status: 1,
+        cumulativeBlockGasUsed: 1n,
+        bitvector: logsBloom,
+        logs: logs,
+    };
+
+    const encodedReceipt = encodeReceipt(receipt, TransactionType.Legacy);
+
+    await trie.put(encodedReceipt, encodedReceipt);
+
+    const proof = await trie.createProof(encodedReceipt);
+    const root = trie.root();
+
+    // Hack to get correct Vec<Bytes>
+    const transformedProof = [[]];
+    for (let i = 0; i < proof.length; i++) {
+        for (let j = 0; j < proof[i].length; j++) {
+            transformedProof[i][j] = proof[i][j];
+        }
+    }
+
+    return { root: root, proof: transformedProof };
```

Comment:
Can we test it with the test that is disabled now? https://github.com/moondance-labs/tanssi/blob/9e682c00ee0ee71fbd4eb63e785e43d885e4f7a0/test/suites/dev-tanssi-relay/ethereum-token-transfers/test_transfer_native_token.ts#L114

Reply:
Sure.

Reply:
Done.